ipu-plugin/pkg/utils/utils.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 Intel Corporation.  All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2024 Intel Corporation.  All Rights Reserved.
```

Reply:
copyrights were added in 2023. I don't think we need to update them on each year.